src/components/NewCommentForm.tsx

```diff
@@ -1,8 +1,77 @@
-import React from 'react';
+import classNames from 'classnames';
+import React, { useContext, useState } from 'react';
+import { client } from '../utils/fetchClient';
+import { GlobalContext } from '../GlobalContext';
+import { Comment } from '../types/Comment';
 
 export const NewCommentForm: React.FC = () => {
+  const {
+    comments,
+    setComments,
+    selectedPost,
+    setIsErrorComments,
+  } = useContext(GlobalContext);
+
+  const [name, setName] = useState('');
+  const [email, setEmail] = useState('');
+  const [text, setText] = useState('');
+  const [inputNameError, setInputNameError] = useState(false);
+  const [inputEmailError, setInputEmailError] = useState(false);
+  const [textAreaError, setTextEreaError] = useState(false);
+  const [isLoadingButton, setIsLoadingButton] = useState(false);
```

Comment:
Rename all boolean variables using [tips](https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig). Fix all cases